src/containers/donation/index.css

```diff
@@ -0,0 +1,171 @@
+body {
+  width: 100%;
+}
+
+.subTitle,
+.firstParagraphStyle,
+.secondParagraphStyle,
+.thirdParagraphStyle,
+.countNumbers,
+.darkGreyColumn,
+.lightGreyColumn {
+  font-family: Roboto, Sans-serif;
+}
+
+.title,
+.secondParagraphStyle h3,
```

Comment:
that being said, similar here, see if you can collapse the below. 

finally, please don't use names like first second thirdPargraph etc, it should be based on their function (see my comments on gunsus about PR for example, about translation names). for example, firstParagraph could instead be .introParagraph, third could be .donationParagraph, etc (these are more useful names, i'm making them up because i can't remember what 1-3rd paragraph are which is exactly why the names are unclear)

Reply:
I can see that most of my issues is about classes, I searched on Gunsu's PR but I didn't find you wrote something about it